apps/backend/src/services/cache_service.py

```
@@ -0,0 +1,503 @@
+"""
+Cache Metadata Service
+
+Service layer for managing cache metadata operations,
+providing business logic for infrastructure data caching management.
+"""
+
+import logging
+from datetime import datetime, timedelta, timezone
+from typing import List, Optional
```

Ruff (0.12.2)

`typing.List` is deprecated, use `list` instead

(UP035)